src/pages/Onboarding/Onboarding.vue

```diff
@@ -5,93 +5,75 @@
       <main class="main-content-container">
         <section id="welcome">
           <div class="cube-div">
-            <img src="./../../assets/img/onboarding/identity-card.svg" class="large-copy-2">
+            <img src="@/assets/img/onboarding/identity-card.svg" class="large-copy-2">
           </div>
           <div class="content">
             <h1>Welcome to Base æpp</h1>
             <p>
-              æternity gives you easy access to the magic of blockchain technology.
-              Follow these quick steps or just jump in.
+              æternity gives you easy access to the magic of blockchain technology. Follow these quick steps or just jump in.
```

Comment:
I don't see a reason to write it in a single line. Will be better to update wording: "Welcome to **the b**ase æpp", "Follow these ~~quick~~ steps or just jump in."